public/app/features/alerting/unified/RuleList.test.tsx

```diff
@@ -603,9 +612,14 @@ describe('RuleList', () => {
 
     testCase('rename just the lotex group', async () => {
       // make changes to form
-      await userEvent.clear(ui.editGroupModal.ruleGroupInput.get());
-      await userEvent.type(ui.editGroupModal.ruleGroupInput.get(), 'super group');
-      await userEvent.type(ui.editGroupModal.intervalInput.get(), '5m');
+      await userEvent.clear(screen.getByRole('textbox', { name: 'Evaluation group', exact: true }));
+      await userEvent.type(screen.getByRole('textbox', { name: 'Evaluation group', exact: true }), 'super group');
+      await userEvent.type(
+        screen.getByRole('textbox', {
+          name: /rule group evaluation interval evaluation interval should be smaller or equal than for values for existing rules in this group\./i,
+        }),
```

Comment:
Same as above!

Reply:
this is a test, we'd better check in the components texts